internal/services/paloalto/next_generation_firewall_vhub_local_rulestack_resource.go

```diff
@@ -276,6 +280,12 @@ func (r NextGenerationFirewallVHubLocalRuleStackResource) Update() sdk.ResourceF
 				firewall.Tags = tags.Expand(model.Tags)
 			}
 
+			if metadata.ResourceData.HasChange("rulestack_id") {
+				ruleStackID, _ := localrulestacks.ParseLocalRulestackID(model.RuleStackId)
```

Comment:
The Line 252 above has checked this error already.

Reply:
it will only get checked there if there is the rulestack_id has been updated. I think we should check it here and in next_generation_firewall_vnet_local_rulestack_resource.go line 280 as well just to be on the safe side.

Reply:
error check added.